ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -93,97 +100,133 @@ export function getSeverityStyle(severity?: Severity) {
       };
     default:
       return {
-        background: BackgroundColor.infoMuted,
+        background: BackgroundColor.backgroundDefault,
         icon: IconColor.infoDefault,
       };
   }
 }
 
-function AlertHeader({
-  selectedAlert,
-  customAlertTitle,
+export function AlertProvider({
+  provider,
+  paddingTop = 0,
+  textAlign,
 }: {
-  selectedAlert: Alert;
-  customAlertTitle?: string;
+  provider?: SecurityProvider;
+  paddingTop?: SizeNumber;
+  textAlign?: TextAlign;
 }) {
   const t = useI18nContext();
-  const severityStyle = getSeverityStyle(selectedAlert.severity);
+
+  if (!provider) {
+    return null;
+  }
+
   return (
-    <>
-      <Box
-        gap={3}
-        display={Display.Block}
+    <Box paddingTop={paddingTop} textAlign={textAlign}>
+      <Text
+        marginTop={1}
+        display={Display.InlineFlex}
         alignItems={AlignItems.center}
-        textAlign={TextAlign.Center}
-        marginTop={3}
+        color={TextColor.textAlternative}
+        variant={TextVariant.bodySm}
       >
         <Icon
-          name={
-            selectedAlert.severity === Severity.Info
-              ? IconName.Info
-              : IconName.Danger
-          }
-          size={IconSize.Xl}
-          color={severityStyle.icon}
+          className="disclosure__summary--icon"
+          color={IconColor.primaryDefault}
+          name={IconName.SecurityTick}
+          size={IconSize.Sm}
+          marginInlineEnd={1}
         />
-        <Text
-          variant={TextVariant.headingSm}
-          color={TextColor.inherit}
-          marginTop={3}
-          marginBottom={4}
-        >
-          {customAlertTitle ?? selectedAlert.reason ?? t('alert')}
-        </Text>
-      </Box>
-    </>
+        {t('securityProviderPoweredBy', [
+          <ButtonLink
+            key={`security-provider-button-link-${provider}`}
+            size={ButtonLinkSize.Inherit}
+            href={SECURITY_PROVIDER_CONFIG[provider]?.url}
+            externalLink
+          >
+            {t(SECURITY_PROVIDER_CONFIG[provider]?.tKeyName)}
+          </ButtonLink>,
+        ])}
+      </Text>
+    </Box>
+  );
+}
+
+function AlertHeader({ severity }: { severity: Severity }) {
```

Comment:
[suggestion] I'm not sure this is better as it's own component. Initially I would think AlertHeader may include the icon with the text, however, it is only the icon.  IMO I think moving the UI 
```
<Icon
      name={severity === Severity.Info ? IconName.Info : IconName.Danger}
      size={IconSize.Xl}
      color={severityStyle.icon}
    />
```
into its own component, <AlertHeader>, does not improve readability since it's minimal code and there is room for interpretation when reading "AlertHeader". Possibly we move the <Icon> directly into <ModalHeader> or rename "AlertHeader" to "AlertHeaderIcon"

Reply:
I reverted my changes to aggregate the icon and the text into the `AlertHeader`.